src/content/community/releases/_index.en.md

```diff
@@ -5,6 +5,15 @@ weight = 40
 +++
 <!-- markdownlint-disable no-duplicate-header -->
 
+## v0.15.1
+
+This is a bugfix release:
+
+* Fixed issue which caused the IPsec pluto process to crash when the remote endpoint was unstable.
+* Fixed issue where a Gateway pod deletion caused data path disruption..
```

Comment:
Duplicate period at the end. 

Not trying to nitpick but I still think "restarted" is more appropriate. We delete pods in testing to effect a restart but I don't think an end user would do that. From the PR https://github.com/submariner-io/submariner/pull/2499, the scenario was due to a gateway failover restart, not explicit deletion of a pod.